modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/store/FilePageStoreManager.java

```diff
@@ -251,10 +237,10 @@ public FilePageStore getStore(int grpId, int partId) throws IgniteInternalChecke
      * @param cleanFiles Delete files.
      */
     void stopAllGroupFilePageStores(boolean cleanFiles) {
-        List<GroupPageStoreHolder<FilePageStore>> holders = new ArrayList<>(groupPageStoreHolders.size());
+        List<List<FilePageStore>> holders = new ArrayList<>(groupPageStoreHolders.size());
 
-        for (Iterator<GroupPageStoreHolder<FilePageStore>> it = groupPageStoreHolders.values().iterator(); it.hasNext(); ) {
-            GroupPageStoreHolder<FilePageStore> holder = it.next();
+        for (Iterator<List<FilePageStore>> it = groupPageStoreHolders.values().iterator(); it.hasNext(); ) {
```

Comment:
Is this method only called on node stop? If yes, can we simply copy the values of `groupPageStoreHolders` and then call `clear`?

Reply:
fix it